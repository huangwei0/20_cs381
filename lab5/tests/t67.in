findall(Z, grandmother('Queen Elizabeth II', Z), Y), 
print(Y), 
print('\n'), 
halt.
