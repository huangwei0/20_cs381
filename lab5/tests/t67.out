[Prince William: Duke of Cambridge,Prince Henry of Wales,Princess Beatrice of York,Princess Eugenie of York,Peter Phillips,Zara Phillips,Lady Louise Windsor,James: Viscount Severn]
